qa/suites/upgrade/jewel-x/point-to-point-x/distros/ubuntu_14.04.yaml

```diff
@@ -1 +0,0 @@
-../../../../../distros/all/ubuntu_14.04.yaml
```

Comment:
Might want to call out in the commit message that Ubuntu is changing here as well

Reply:
oh forgot that :(

Reply:
Jewel is only tested on 14.04, so the impact of this change is unknown.

Reply:
@smithfarm shall we test or undo?

Reply:
Ubuntu 14.04 is not going away, so I don't see why we need to migrate jewel away from it.